### PR TITLE
Add LICENSE file

### PR DESCRIPTION
**Pull request by rayzhou2017**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (pixiake) [inline LICENSE:189]: Whether the information about this project need to be  modified？ Such as: `Copyright 2020 Kubekey `

Comment (rayzhou2017) [reply LICENSE]: fixed

**Pull request merged**